Sources/SwiftSyntax/generated/syntaxNodes/SyntaxNodesC.swift

```diff
@@ -1635,16 +1628,16 @@ public struct ClosureCaptureSyntax: SyntaxProtocol, SyntaxHashable, _LeafSyntaxN
   /// ### Tokens
   /// 
   /// For syntax trees generated by the parser, this is guaranteed to be `<identifier>`.
-  public var name: TokenSyntax? {
+  public var name: TokenSyntax {
```

Comment:
Changing `name` to be non-optional is an API-breaking change because eg. `closuereCapture.name?.text` will stop compiling. There is no way of making it API-compatible. Could you highlight it in the release notes?

Reply:
Thanks for pointing this out. I added a note under "API-Incompatible Changes" in the release notes.